ci/run.yml

```
@@ -0,0 +1,25 @@
+- hosts: controller
+  vars:
+    fvt_dir: "{{ ansible_env.HOME}}/{{ zuul.projects['github.com/infrawatch/feature-verification-tests'].src_dir }}"
+  tasks:
+    - name: Install ansible so that ansible-playbook is available
+      ansible.builtin.pip:
+        name:
+          - ansible
+
+    - name: "Run a playbook locally on the target host"
+      ansible.builtin.shell:
+        cmd: |
+          ANSIBLE_CONFIG=ci/ansible.cfg ansible-playbook -v ci/stf_tests.yml
+        chdir: "{{ fvt_dir }}"
+      register: output
```

Okay, I think the issue is that the cifmw vars are available in teh context of this run playbook. But the stf_tests.yml playbook is completely isolated from the vars and facts in this context.

We need to pass in the vars we want.
In this case, cifmw_path and cifmw_kubeconfig.

You could do the login here (i.e. ci.run) and then pass in the KUBECONFIG env var to this task. (option 1)
```suggestion
    - name: "Run a playbook locally on the target host"
      ansible.builtin.shell:
        cmd: |
          ANSIBLE_CONFIG=ci/ansible.cfg ansible-playbook -v ci/stf_tests.yml
        chdir: "{{ fvt_dir }}"
      environment:
          KUBECONFIG: "{{ cifmw_openshift_kubeconfig }}"
          PATH: "{{ cifmw_path }}"
      register: output
```
You could also pass in all the required vars here. and do the login in stf_tests. (option 2)

However, I would recommend option 1. We can then use the feature-verification tests assuming that we're already logged in, and then we can keep the cifmw vars separate form the role config vars.